test/cache-get.test.ts

```diff
@@ -1,19 +1,21 @@
 import * as CacheGet from '../src/messages/responses/cache-get';
-import {Buffer} from 'node:buffer';
+import {TextEncoder} from 'util';
+
+const TEXT_ENCODER = new TextEncoder();
 
 describe('CacheGet', () => {
   describe('#toString()', () => {
     it('shows a short value', () => {
       const value = 'Napoleon';
-      const hit = new CacheGet.Hit(Buffer.from(value));
+      const hit = new CacheGet.Hit(TEXT_ENCODER.encode(value));
```

Comment:
Why the switch from buffer to text encoder?

Reply:
Node 10 and 12 don't have it. Now that we're dropping Node 10 and 12 maybe we should switch from TextEncoder to Buffer. However, TextEncoder is what we use everywhere else, and I don't know the benefits/drawbacks of Buffer vs TextEncoder, so switching is a bigger change.

Reply:
okay